documentation/weave/index.md

```
@@ -37,52 +37,76 @@ updated: "2024-12-12"
-  - global-copy-strategy: overwrite | no-overwrite | skip | prompt
-  - per-inclusion copy-strategy: overwrite | no-overwrite | skip | prompt
-- weave watch: detects changes in active inclusions and copies them to dest
-  - ?does it
-- weave start: build and watch,
-  - ?but only safely (i.e., repos all up-to-date, no collisions, build with
-    prompt)
-
+-   **weave setup**: interactive prompt to create config file if none present, and add inclusions
+-   **weave inclusions list**: lists all inclusions and their statuses (active/inactive, present/missing, current/ahead/behind/divergent) and copy strategies, in order but grouped by active/inactive
+-   **weave inclusions verify**: output status "ready to weave", "not ready"
+    -   maybe suggest `--ignore-missing` and `repos prepare` if needed
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)